util/design/secded_gen.py

```diff
@@ -76,6 +78,9 @@
 # secded configurations
 SECDED_CFG_FILE = "util/design/data/secded_cfg.hjson"
 
+PROJ_ROOT = Path(__file__).parent.joinpath('../../')
+SECDED_CFG_PATH = Path(PROJ_ROOT).joinpath(SECDED_CFG_FILE)
```

Comment:
nit: you could also do `SECDED_CFG_PATH = Path(PROJ_ROOT) / SECDED_CFG_FILE`

Reply:
ahhh. let me do that.